src/app/leader-board/leader-board.component.html

```diff
@@ -0,0 +1,23 @@
+<div id="body">
+  <div id="close" (click)="close()">&times;</div>
+  <img [src]="image" [matTooltip]="language?.badge + badgeName">
+  <br>
+  <br>
+  <table class="table table-striped">
+    <thead>
+    <tr>
+      <th>{{language?.rank}}</th>
+      <th>{{language?.name}}</th>
+      <th>{{language?.points}}</th>
+    </tr>
+    </thead>
+    <tbody *ngFor="let user of data; index as i">
+    <tr>
+      <td [ngClass] = "{ 'user' : user.userEmail === userEmail } ">{{i + 1}}</td>
```

Comment:
Here we are using the **user** from the **data** array in the For loop. It becomes convenient to check the condition of active user, while we are binding it in the template. 
It can be moved to the .ts file, then I check for this condition and associate some value to a some new variable, and then I again check equality of the variable, while binding it. 
So some condition needs to checked in the template in either case, as we want to control styling of the particular division.

Reply:
Can't we have a function that takes it as a parameter in the ts file and returns the appropriate value?

Reply:
I think what @Raxerz is suggesting will work and is a good way.

Reply:
Okay! Got it. Done.